internal/provider/file/file_test.go

```diff
@@ -24,7 +29,7 @@ var (
 			Provider: "file",
 			Image: model.Image{
 				Name:      "docker.bintray.io/jfrog/xray-server:2.8.6",
-				WatchRepo: true,
+				WatchRepo: utl.NewTrue(),
```

Comment:
Same here and others

```suggestion
				WatchRepo: utl.NewFalse(),
```